csharp/tests/Valkey.Glide.UnitTests/CommandTests.cs

```diff
@@ -37,6 +55,64 @@ public void ValidateCommandConverters()
             () => Assert.Equal<GlideString>("OK", Request.Get("a").Converter("OK")),
             () => Assert.Null(Request.Get("a").Converter(null)),
             () => Assert.Equal("info", Request.Info([]).Converter("info")),
+
+            () => Assert.True(Request.SetAddAsync("key", "member").Converter(1L)),
+            () => Assert.False(Request.SetAddAsync("key", "member").Converter(0L)),
+            () => Assert.True(Request.SetRemoveAsync("key", "member").Converter(1L)),
+            () => Assert.False(Request.SetRemoveAsync("key", "member").Converter(0L)),
+
+            () => Assert.Equal(2L, Request.SetAddAsync("key", ["member1", "member2"]).Converter(2L)),
+            () => Assert.Equal(1L, Request.SetRemoveAsync("key", ["member1", "member2"]).Converter(1L)),
+            () => Assert.Equal(5L, Request.SetLengthAsync("key").Converter(5L)),
+            () => Assert.Equal(3L, Request.SetIntersectionLengthAsync(["key1", "key2"]).Converter(3L)),
+            () => Assert.Equal(4L, Request.SetUnionStoreAsync("dest", ["key1", "key2"]).Converter(4L)),
+            () => Assert.Equal(2L, Request.SetIntersectStoreAsync("dest", ["key1", "key2"]).Converter(2L)),
+            () => Assert.Equal(1L, Request.SetDifferenceStoreAsync("dest", ["key1", "key2"]).Converter(1L)),
+
+            () => Assert.Equal<GlideString>("member", Request.SetPopAsync("key").Converter("member")),
+            () => Assert.Null(Request.SetPopAsync("key").Converter(null)),
+        ]);
+    }
+
+    [Fact]
+    public void ValidateSetCommandHashSetConverters()
+    {
+        var testHashSet = new HashSet<object> { 
+            (GlideString)"member1", 
```

Comment:
hint: you can use `gs`:
```suggestion
            (gs)"member1",
```